src/braket/pennylane_plugin/ops.py

```diff
@@ -497,3 +480,7 @@ def compute_matrix(phi_0, phi_1, theta):
     def adjoint(self):
         (phi_0, phi_1, theta) = self.parameters
         return AAMS(phi_0 + np.pi, phi_1, theta, wires=self.wires)
+
+
+def _cast_to_tf(val):

```

Comment:
`qml.math.get_interface` can take in multiple values. For multi angle gates, should we pass in iterables?